### ISSUE TITLE
Small blemish | Rank by css will displayed as list item in the profile

### ISSUE THREAD

**Issue by rich20** [Kunena/Kunena-Forum#6588]
You can see it on the image. Left is Rank by css and right Rank by image.

--------------
![g1123](https://user-images.githubusercontent.com/888847/68597491-57f0d900-049d-11ea-8b39-29f874bb232a.png)

**Comment by Sophist-UK** [comment]
yes - looks like an un-ordered list bullet to me (which coincidentally just happens to be a smaller version of the graphic content of the list).
